targets/drive/web/services.jsx

```diff
@@ -41,13 +42,14 @@ document.addEventListener('DOMContentLoaded', () => {
 
   const client = new CozyClient({
     cozyURL: cozyUrl,
-    token: data.cozyToken,
-    offline: { doctypes: ['io.cozy.files'] }
+    token: data.cozyToken
```

Comment:
Since in the service, we fetch via cozy.client.fetchDocument we had to deactivate offline otherwise the adapter for `io.cozy.files` was not the right one (pouchdbAdapter) and we had a 404. Is this a problem @goldoraf ?

Reply:
It seems legit, the service doesn't need offline support. But see my above comment